linkerd/app/src/lib.rs

```diff
@@ -339,31 +338,31 @@ impl App {
                         }
 
                         // Spawn the DNS resolver background task.
-                        tokio_02::spawn(dns.instrument(info_span!("dns")));
+                        tokio::spawn(dns.instrument(info_span!("dns")));
 
                         if let tap::Tap::Enabled { daemon, serve, .. } = tap {
-                            tokio_02::spawn(daemon.instrument(info_span!("tap")));
-                            tokio_02::task::spawn(
+                            tokio::spawn(daemon.instrument(info_span!("tap")));
+                            tokio::task::spawn(
```

Comment:
nit: should probably be 
```suggestion
                            tokio::spawn(
```
for consistency (I think this was a `spawn_local` at one point or something)